dts/bindings/adc/nxp,lpc-lpadc.yaml

```diff
@@ -75,6 +75,12 @@ properties:
     type: phandles
```

Comment:
look at the history on the file for some hints on the syntax to use. 

dts: bindings: lpadc: Add new ref-supply-val property

Reply:
ok, I will update the commit message, make it more syntax-compliant.